src/commands/utility/changeStatus.js

```diff
@@ -48,19 +48,19 @@ module.exports = {
         );
       }
 
-      let channelName = channel.name;
-      Object.values(MAPPED_STATUS_COMMANDS).forEach((emoji) => {
-        if (channelName.startsWith(emoji)) {
-          channelName = channelName.replace(emoji, '').trim();
-        }
-      });
+      const escapedEmojis = Object.values(MAPPED_STATUS_COMMANDS)
+        .map((emoji) => emoji.replace(/[.*+?^${}()|[\]\\]/g, '\\$&'))
+        .join('|');
+      const emojisRegExp = new RegExp(`^(${escapedEmojis})\\s?`, 'i');
```

Comment:
not sure if this last part is needed, please explain if it is needed 👀 
```suggestion
      const emojisRegExp = new RegExp(`^(${escapedEmojis})`, 'i');
```

Reply:
completed